src/capture/capture_video.cpp

```diff
@@ -413,7 +413,7 @@ void capture_video_add_frame(const RenderedImage& image, const float frames_per_
 				}
 				break;
 
-			case PixelFormat::XRGB8888_Packed32:
+			case PixelFormat::BGR32_ByteArray:
```

Comment:
Has video capture been tested?  If I recall correctly, ZMBV expects a packed 32 bit format in host native endianess.  Since you've changed this to a byte array, we might need to change some logic here.

Reply:
Yes, video capture in paletted and 15/16/24/32-bit hi/true colour modes need to be tested as well.

Reply:
Those are working with the changes so far, but I've only been able to hit the non-double-width code path for the high colour modes. 
 
@johnnovak, in capture_video.cpp's `capture_video_add_frame` function, do you know which games/program can get me into the double-width conditional?

```c++
for (auto i = 0; i < src.height; ++i) {
    const uint8_t* row_pointer = row_buffer;
    if (src.double_width) {
        switch (src.pixel_format) {
            ...
            case PixelFormat::RGB555_Packed16:
            case PixelFormat::RGB565_Packed16:
```

(So far I'm only getting the 8-bit and below modes into it)